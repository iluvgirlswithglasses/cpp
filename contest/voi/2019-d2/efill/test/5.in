38
-4058222 -3028654
-4058066 -3847216
-4058018 -3915758
-4052598 -4038858
-4051336 -4050334
-3962372 -4058188
1192344 -4058230
3983370 -4057920
4017928 -4057090
4041596 -4033998
4054068 -3985352
4056426 -3922696
4056846 -3849790
4057506 -3650126
4057932 -3154720
4058130 -2914542
4058148 -2555666
4058052 1317284
4057952 3783700
4057672 3844878
4056684 3989570
4051888 4009188
4023782 4053722
3995652 4056640
3937548 4057836
3481102 4058002
1173988 4058206
-334890 4058238
-704836 4058242
-2646948 4058238
-3415668 4058096
-3591586 4057954
-3826166 4057560
-4025284 4056600
-4035166 4056078
-4053916 4009444
-4056612 3937632
-4058208 3880694
-4057884 -4802205 -4058326 -2482946
