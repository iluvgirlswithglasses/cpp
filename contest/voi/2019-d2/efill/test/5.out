26961514.73
